src/tablo/Cell/Cell.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import t, { maybe, enums, union } from 'tcomb';
+import { props  } from 'tcomb-react';
```

Comment:
I'm fixing it now, this was *quite hard* in a way.

Why don't we have roots modules in brc?